README.md

```diff
@@ -17,10 +17,10 @@ calculate the best path to take to avoid the pitfalls; and collect bonus points.
 The cars move autonomously on the screen within the race track game with no interference
 from the students. No joystick or mouse would be used.
  
-In order to control the car movements, the students needs to implement a 'driver'.
+In order to control the car movements, the student needs to implement a 'driver'.
```

Comment:
I don't think this change is better, we have many students implementing a driver code.